source/ptreferential/ptreferential_api.cpp

```diff
@@ -50,7 +50,8 @@ static void extract_data(navitia::PbCreator& pb_creator,
                          const type::Indexes& rows,
                          const int depth) {
     pb_creator.action_period = data.meta->production_period();
-    const auto with_line_sections = DumpMessageOptions{DumpMessage::Yes, DumpLineSectionMessage::Yes};
+    auto with_line_sections = DumpMessageOptions{DumpMessage::Yes, DumpLineSectionMessage::Yes};
```

Comment:
Is it necessary to remove const ?

Reply:
yes because of https://github.com/CanalTP/navitia/pull/3466/files#diff-e83410d27d0f652f3ceeeb42b7900afdcc97a985ff19b64251048fca334e6fecR54